tests/pipelines/fgcmOutputProductsLatiss.yaml

```diff
@@ -0,0 +1,5 @@
+description: Output fgcmcal products
+instrument: lsst.obs.lsst.Latiss
+tasks:
+  fgcmOutputProducts:
+    class: lsst.fgcmcal.fgcmOutputProducts.FgcmOutputProductsTask
```

Comment:
It looks like all these pipelines differ from the HSC versions only by the imported `instrument`.  Would making a bas e (or `ingredients` file?) make sense/be worth it?

Reply:
Maybe when I add a third test camera, but with 2 it's definitely not worth it (yet) to make a whole new directory structure just for test configs.